readme.md

```diff
@@ -53,7 +53,10 @@ function Counter() {
   return (
     <h1>
       {count}
-      <button onClick={() => setCount(c => c + 1)}>one up</button>
+      <button onClick={() => setCount((c) => c + 1)}>one up</button>
+    </h1>
+  )
+}
```

Comment:
```suggestion
      <button onClick={() => setCount((c) => c + 1)}>one up</button>
      ...
```

How about adding `...` to make it explicit that it's partial code?

Reply:
good idea~  ， It works

Reply:
Shall I modify it and commit merge request  again?

Reply:
Yeah, please modify entirely with the said policy.

Reply:
Got it~！